lib/Dialect/Calyx/Export/CalyxEmitter.cpp

```diff
@@ -522,7 +523,16 @@ void Emitter::emitGroup(GroupOp group) {
           });
     }
   };
-  emitCalyxSection("group", emitGroupBody, group.sym_name());
+  auto prefix =
```

Comment:
nit (feel free to ignore): Is TypeSwitch necessary here? Seems to produce more code than necessary.

Reply:
Not strictly necessary; in general i like having an error if/when a new class is added to an interface but support is yet to be added. In the given case, however, we're probably not expecting a bunch of new group ops to be added - I can remove it if you prefer.

Reply:
Yeah that was my thought process. A one liner for this may be
```
StringRef prefix = isa<CombGroupOp>(op) ? "comb group" : "group";
```

However, do as you please. This is mostly just me nitpicking.